concent_api/core/tests/test_unit_validation.py

```diff
@@ -1,7 +1,9 @@
-from unittest import TestCase
```

Comment:
This change is not necessary and I think also against our coding style.

Reply:
In this module both `unittest.TestCase` and `django.test.TestCase` are used

Reply:
Just for other reviewers, you could always used `import X as Y`. I know you know that ;)

Reply:
You could, but most of the time I prefer the way @dybi did it. Unless the new name is very close to the old one, renaming makes it harder to find it with normal text search and is confusing when you're reading just a part of the code rather than the whole file from the beginning to end.